code/_onclick/hud/robot.dm

```diff
@@ -1,241 +1,241 @@
-/obj/screen/robot
```

Comment:
This will still need to be addressed; I recommended deleting this file, outright, then downloading it directly off Para's git website and putting it back in to see if it resolves this.